### DISCUSSION TITLE
Lively wallpaper methord to have slide show of live wallpapers

### DISCUSSION THREAD

**Discussion by Shurikensenpai** [rocksdanister/lively#1044] [Ideas]
We all appreciate the things which lively wallpaper has made and done for us but we want more as we are a demanding public i would like to raise the idea of slideshow method with live wallpapers which will let us get out of the same boring wallpaper and  give us something new even tho its old we still can get the feel of it

**Comment by rocksdanister** [comment]
Its under development https://github.com/rocksdanister/lively/issues/137
